### DISCUSSION TITLE
ArgumentNullException error

### DISCUSSION THREAD

**Discussion by Unreal852** [realm/realm-dotnet#3093] [Q&A] [answered]
Hello, i'm making an application with AvaloniaUI and i decided to use Realm as my database but when i bind my view to fields that belongs to an IRealmObject it throws ArgumentNullException. I created a separate project to replicate the issue https://github.com/Unreal852/RealmAvalonia
I did not create an issue because i don't know if i'm doing things wrong or not.

**Comment by papafe** [comment, marked as answer]
Hey @Unreal852, thanks for your message. I have created issue #3094 from this so we can keep track of what's happening with the investigation.
Also, it would be good if you specify exactly when that exception is being raised.

**Reply by Unreal852** [reply]
Hello, i gave more details on the issue https://github.com/realm/realm-dotnet/issues/3094#issuecomment-1313778136
